Modules/EMCAL/include/EMCAL/DigitsQcTask.h

```diff
@@ -95,9 +106,12 @@ class DigitsQcTask final : public TaskInterface
   void endOfActivity(Activity& activity) override;
   void reset() override;
 
-  void setThreshold(Double_t threshold) { mCellThreshold = threshold; }
   void setEndOfPayloadCheck(Bool_t doCheck) { mDoEndOfPayloadCheck = doCheck; }
 
+  bool hasConfigValue(const std::string_view key);
+  std::string getConfigValue(const std::string_view key);
+  std::string getConfigValueLower(const std::string_view key);
```

Comment:
@knopers8 @Barthelemy Could those functions be pushed upwards to the TaskInterface?

Actually I am thinking about something similar to what is done for the Options by the framework having variant data types for int, float, bool and string, and a template function casting to the required type. (Would remove the need for the lambdas I am using to extract bool and float from the string myself)

Reply:
I created a ticket: https://alice.its.cern.ch/jira/browse/QC-632